packages/datagateway-download/cypress/integration/adminDownloadStatus.spec.ts

```diff
@@ -0,0 +1,157 @@
+      'have.text',
+      '675eea67-bad7-4209-b54e-87f0d97eb50d'
+    );
+  });
+
+  describe('should be able to sort download items by', () => {
+    it('ascending order', () => {
+      cy.get('.react-draggable')
+        .eq(1)
+        .trigger('mousedown')
+        .trigger('mousemove', { clientX: 300 })
+        .trigger('mouseup');
+      cy.contains('[role="button"]', 'Prepared ID').click();
+
+      cy.get('[aria-sort="ascending"]').should('exist');
+      cy.get('.MuiTableSortLabel-iconDirectionAsc').should('be.visible');
+      cy.get('[aria-rowindex="1"] [aria-colindex="2"]').should(
+        'have.text',
+        '000137b1-9f20-4731-b48c-58a6effdd060'
+      );
+    });
+
+    it('descending order', () => {
+      cy.get('.react-draggable')
+        .eq(1)
+        .trigger('mousedown')
+        .trigger('mousemove', { clientX: 300 })
+        .trigger('mouseup');
+      cy.contains('[role="button"]', 'Prepared ID').click();
+      cy.contains('[role="button"]', 'Prepared ID').click();
+
+      cy.get('[aria-sort="descending"]').should('exist');
+      cy.get('.MuiTableSortLabel-iconDirectionDesc').should(
+        'not.have.css',
+        'opacity',
+        '0'
+      );
+      cy.get('[aria-rowindex="1"] [aria-colindex="2"]').should(
+        'have.text',
+        'fffc2df5-bf1e-4403-8a9f-b75f1a0a4d57'
+      );
+    });
+
+    it('no order', () => {
+      cy.get('.react-draggable')
+        .eq(1)
+        .trigger('mousedown')
+        .trigger('mousemove', { clientX: 300 })
+        .trigger('mouseup');
+      cy.contains('[role="button"]', 'Prepared ID').click();
+      cy.contains('[role="button"]', 'Prepared ID').click();
+      cy.contains('[role="button"]', 'Prepared ID').click();
+
+      cy.get('[aria-sort="ascending"]').should('not.exist');
+      cy.get('.MuiTableSortLabel-iconDirectionDesc').should('not.exist');
+      cy.get('.MuiTableSortLabel-iconDirectionAsc').should(
+        'have.css',
+        'opacity',
+        '0'
+      );
+      cy.get('[aria-rowindex="1"] [aria-colindex="2"]').should(
+        'have.text',
+        '675eea67-bad7-4209-b54e-87f0d97eb50d'
+      );
+    });
+
+    it('multiple columns', () => {
+      cy.contains('[role="button"]', 'Username').click();
+      cy.contains('[role="button"]', 'Prepared ID').click();
+
+      cy.get('[aria-rowindex="1"] [aria-colindex="2"]').should(
+        'have.text',
+        '06bb426b-845f-486a-b5eb-10bfb7e8fb10'
+      );
+    });
+  });
+
+  describe('should be able to filter download items by', () => {
+    it('text', () => {
+      cy.get('[aria-label="Filter by Prepared ID"]')
+        .find('input')
+        .first()
+        .type('675eea67');
+
+      cy.get('[aria-rowcount="1"]').should('exist');
+      cy.get('[aria-rowindex="1"] [aria-colindex="2"]').should(
+        'have.text',
+        '675eea67-bad7-4209-b54e-87f0d97eb50d'
+      );
+    });
+
+    it.skip('date between', () => {
+      // TODO: Test Requested Date fukter
```

Comment:
Ignore the spelling mistake but the _Requested Date_ column filter should be tested - I didn't have time. In addition to this, it would also be good to test the action buttons.